src/Sarif.Viewer.VisualStudio.Core/Sarif/StackFrame.Extensions.cs

```diff
@@ -12,7 +12,7 @@ public static StackFrameModel ToStackFrameModel(this StackFrame stackFrame, int
         {
             var model = new StackFrameModel(resultId, runIndex);
 
-            model.FullyQualifiedLogicalName = stackFrame.Location.LogicalLocation.FullyQualifiedName;
+            model.FullyQualifiedLogicalName = stackFrame.Location?.LogicalLocation?.FullyQualifiedName;
```

Comment:
Do we have a test covering this nullable parts? If not, pls, add.

Reply:
found the issue by new added SarifErrorListItemTests, also added unit tests for StackFrameExtensions.ToStackFrameModel itself.